docs/references/_attachments/ic.did

```diff
@@ -302,7 +302,9 @@ type http_request_args = record {
     headers : vec http_header;
     body : opt blob;
     transform : opt record {
-        function : func(record { response : http_request_result; context : blob }) -> (http_request_result) query;
+        function : func(record { response : http_request_result; context : blob };
+    is_replicated : opt bool;
+) -> (http_request_result) query;
```

Comment:
this is syntactically wrong, the new field should not be part of the transform

Reply:
🙈 apologies, fixed now